contracts/governance/extensions/GovernorTimelockControl.sol

```diff
@@ -64,7 +64,11 @@ abstract contract GovernorTimelockControl is IGovernorTimelock, Governor {
             return ProposalState.Executed;
         } else if (_timelock.isOperationPending(queueid)) {
             return ProposalState.Queued;
+        } else if (_timelock.isOperationDone(queueid)) {
+            // This can happen if the proposal is executed directly on the timelock.
+            return ProposalState.Executed;
```

Comment:
The merge wasn't done properly. `isOperationDone` is already called 2 lines above. Which one should we keep?

Reply:
`isOperationPending` and `isOperationDone` are exclusive, so the order is irrelevant.

There are "public" calls that have a Queued state in their path (execute) but none that have an Executed state in their path. So I would optimize for queued

Reply:
Fixed